src/com.frostycoolslug.goxlr-utility.sdPlugin/app/toggle-mute.js

```diff
@@ -12,6 +11,7 @@ function muteToggleExternalStateChange() {
 }
 
 muteToggle.onKeyDown(({action, event, context, device, payload}) => {
+    let longPressDelay = status.mixers[payload.settings.serial].settings.mute_hold_duration;
```

Comment:
It's definitely worth checking whether `status === undefined` or `status.mixers[payload.settings.serial] === undefined`, the first could occur if there's been a communication loss to the Utility, and the second could occur if the configured device was unplugged, but both would lead to a 'cannot read X of undefined' error.

If either are undefined, probably best to just define a fixed 500ms, if there's an issue when the command gets executed, it'll throw up the warning indicator and carry on as normal.